lua/session_manager/init.lua

```diff
@@ -13,13 +13,26 @@ function session_manager.load_last_session(bang)
   end
 end
 
+function session_manager.load_current_session(bang)
+  local cwd = vim.fn['getcwd']()
+  local session_name = utils.dir_to_session_filename(cwd)
```

Comment:
`vim.fn.cwd` is slow, prefer to use `vim.loop.cwd()`:

```suggestion
  local session_name = utils.dir_to_session_filename(vim.loop.cwd())
```